contrib/ansible/roles/common/tasks/main.yml

```diff
@@ -4,6 +4,16 @@
   register: s
   changed_when: false
 
+- name: Determine if CentOS7
+  shell: "grep 'CentOS Linux.*release 7' /etc/redhat-release"
+  register: is_centos7
+  ignore_errors: yes
+  changed_when: false
+
+- name: Install Testing centos7 repo for new tool versions
+  copy: src=virt7-testing.repo dest=/etc/yum.repos.d/virt7-testing.repo
+  when: is_centos7
```

Comment:
Lets move this repo copy to its own task.  In case other distros want to do stuff I don't want to keep cluttering this file...